docs/benchit_user_guide.md

```diff
@@ -221,11 +221,12 @@ The default usage of `benchit` is to directly provide it with a python script, f
 _Note_: Some of these tasks have to do with the MLAgility `cache`, which stores the `build directories` (see [Build](#build)).
 
 The commands are:
-- `benchmark` (default command): benchmark the model(s) in one or more scripts
-- `list`: list the available builds in the cache
-- `delete`: delete one or more builds from the cache
-- `report`: print a report in .csv format summarizing the results
-- `version`: print the `benchit` version number
+- [`benchmark`](#benchmark-command) (default command): benchmark the model(s) in one or more scripts
+- [`list`](#list-command): list the available builds in the cache
+- [`state`](#state-command): print the [state](https://github.com/groq/groqflow/blob/main/docs/user_guide.md#stateyaml-file) of one or more builds from the cache
```

Comment:
"state" is ok for now, but it is not a very accurate name for that is being done.

In the future we should probably rename this to something along the lines of `list_builds` and `print_build_info`.